api/topo/topo.proto

```diff
@@ -55,6 +55,7 @@ message ReadResponse {
 message SubscribeRequest {
   Reference ref = 1;
   bool withoutReplay = 2;
+  bool snapShot = 3;
```

Comment:
Perhaps it would be better to use  underscore_separated_names for field names (including oneof field and extension names) – for example, song_name based on protobuf style guides.

Reply:
done